src/main/java/br/com/sbk/sbking/networking/kryonet/KryonetClientListenerFactory.java

```diff
@@ -11,7 +12,7 @@
 
 public class KryonetClientListenerFactory {
 
-  public static Listener getClientListener(Client client) {
+  public static Listener getClientListener(BlockingQueue<SBKingMessage> clientMessageQueue) {
```

Comment:
I didn't understand why are you calling this variable here as `clientMessageQueue`, my problem is with the "client". Because in other client classes you aren't calling this variable clientMessageQueue, just `messageQueue`.

Reply:
No specific reason, I thought both were quite as good names.
In fact, in the beginning, I though they would share the same type, but then I realized that the server would need more information (who sent the message) while the client always know that (the server sent the message) so they end up being different. By this point, I figured it would be good to diferentiate them in the variable naming also.